src/phylo/phylogeny.jl

```diff
@@ -0,0 +1,230 @@
+# phylo/phylogeny.jl
+# ==================
+#
+# Types and methods for phylogenetic trees.
+#
+# Part of the Bio.Phylo module.
+#
+# This file is a part of BioJulia. License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+
+
+# The Phylogeny Type
+# -------------------
+
+"""
+Phylogeny represents a phylogenetic tree.
+
+Fields:
+
+    - graph: A LightGraphs directional graph.
```

Comment:
Added a clarification to the doctoring that the graph field is a directional graph. And for phylogenies we use the directional graph as implemented in LightGraphs as the DiGraph type. This should be clearer to readers that this is a type imported from a different package.